### ISSUE TITLE
Migrate Galasa infrastructure onto OMP-supplied IBM cloud account

### ISSUE THREAD

**Issue by techcobweb** [galasa-dev/projectmanagement#1986]
# Story
As a galasa open source owner, (linux foundation), I want the Galasa project to be running entirely from Linux foundation-funded infrastructure, so it's ongoing development is not dependent upon IBM to provide the build test and publish infrastructure.

# Background
- Galasa currently straddles 2 IBM cloud accounts, due to historical reasons.
- We have an IBM cloud account set up and paid for by the Linux foundation.
- We should switch over to using it and retire the other two IBM cloud accounts.

To do this, we have to approach the problem in stages:

# Tasks
- [x] Allocate a kube cluster on which we can run the test system(s)
- [x] Allocate a secrets manager to hold secrets
- [x] Populate the secrets manager with copies of secrets from the CIO account instance.
**>> need to update both if we change anything now<<**
- [x] Create the secrets-manager namespace to install the secrets operator CRD
- [x] Create secret-api-key secret that contains IBM Cloud API Key that links the SecretStore to IBM Cloud
- [x] Create the ibmcloud-secrets-manager SecretStore that links to the new OMP Account
   - [x] not working right now, need to install external-secrets Helm chart
- [x] Move the secrets manager so that all secrets for the OMP infrastructure are sourced from here, or stored in here for posterity.
- [x] Update the build infrastructure so that secrets in are sourced from OMP account IBM Cloud secrets manager:
- [x] Move the ecosystems/deployed Kubernetes services which are used to test and demonstrate the Galasa capabilities onto the OMP-owned IBM cloud account
   - [x] argocd
   - [x] cert-manager
   - [x] galasa-development (development.galasa.dev sites)
   - [x] galasa-ecosystem1 (to be renamed to galasa-service1)
- [x] Ensure triggers.galasa.dev still works
- [x] Move the DNS forwarding capability currently on the CIO IBM cloud account to the OMP-hosted account, with ownership of the galasa.dev DNS domain.
- [x] Raise ticket for LF to create DNS records routing requests to the new VPC load balancer
- [x] Switch old DNS records to new VPC load balancer
- [x] Delete new temporary DNS records
- [x] Move the Event Streams instance
- [x] Move the hosting of the galasa docs web site to github pages if possible - Done in another story.
- [x] Set up CPS props and users on galasa-service1 so we can start using it for testing and decommission galasa-ecosystem1
- [x] Switch URLs in automation to galasa-service1 from galasa-ecosystem1
- ~Move the test infrastructure so that it tests Galasa against 3270 resources supplied by the OMP organisation~ - **moved this task to a separate story as we still don't have an OMP-supplied 3270 resource**

**Comment by techcobweb** [comment]
Marking as blocked because I can't currently allocate any services/resources on the OMP cloud account.

**Comment by techcobweb** [comment]
Hi @jadecarino Adding you to this story so you know what's awaiting your return ;-)
We will need to change the builds to use the new secrets manager.

**Comment by jadecarino** [comment]
Complete this as cleanup when this is done: http://github.com/galasa-dev/projectmanagement/issues/2145

**Comment by jadecarino** [comment]
Currently blocked on setting up any services on the cluster as the cluster is a "Virtual Private Cluster" instead of a "Classic Cluster", meaning it's blocking outbound traffic. No Pods can start up as they can't reach the image registry (dockerhub, OCI) to pull the image from for the Pod. I have been advised to reach out on a Slack channel and open a support ticket for help on this.

**Comment by jadecarino** [comment]
Ticket opened and sent to LF to ask how to proceed for the DNS records.

**Comment by jadecarino** [comment]
Setting back to blocked as I'm unable to create namespaces in the IBM Cloud Container Registry. I have asked John Mertic for advice.
